imagekit/src/test/java/com/imagekit/android/URLGenerationTests.kt

```diff
@@ -222,7 +221,7 @@ class URLGenerationTests {
     @Test
     fun urlConstructionCustomQueryParameterWithTransforms() {
         val expectedTransformation =
-            "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?${ImageKit.IK_VERSION_KEY}=android-${BuildConfig.API_VERSION}&x-test-header=Test&tr=h-300,w-300:rt-90"
+            "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?x-test-header=Test&tr=h-300,w-300:rt-90"
         val actualTransformation =
```

Comment:
Need to have test cases for adaptive bitrate URL generation and in general all new video related parameters that are added now.

Reply:
Implemented in #33